rwiki/rwiki-impl/impl/src/java/uk/ac/cam/caret/sakai/rwiki/component/service/impl/RWikiObjectServiceImpl.java

```diff
@@ -483,7 +484,7 @@ private void update(String name, String user, String realm, Date version,
 					notiPriority));
 			}
 		}
-		catch (HibernateOptimisticLockingFailureException e)
```

Comment:
I'd say you should either keep it for a multiple catch '(HibernateOptimisticLockingFailureException | RollbackException e)' or remove the import if you think it can't happen anymore. Thanks!

Reply:
I can't remove the import because we have another catch here:
https://github.com/sakaiproject/sakai/blob/master/rwiki/rwiki-impl/impl/src/java/uk/ac/cam/caret/sakai/rwiki/component/service/impl/RWikiObjectServiceImpl.java#L557

Maybe this catch should be changed to... but I don't know. Obviously this would be another different problem than the purpose of the jira.

Reply:
Ok, I guess the other one could be replaced too but as we're not sure (and it's not a big deal) let's keep it as it is.